basics/memory.md

```diff
@@ -5,7 +5,7 @@ hantering av minnet. Med det sagt är manuell minneshantering en källa
 till många fel, så D använder sig av en *skräpsamlare* (garbage collector) som
 standardmetod för att hantera minnestilldelningar.
 
-D erbjuder en pekartyp `*T` likt i programmeringsspråket C:
+D erbjuder en pekartyper `T*` likt som i programmeringsspråket C:
```

Comment:
Remove `en`

Reply:
Ah of course, my bad! Shouldn't have commited so late at night.